### PR TITLE
Upgrade RubyMine EAP to 141.373

### PR DESCRIPTION
**Pull request by sebroeder**
- Unify with other IntelliJ-based formulas (see #879)
- Mention formula with bundled JDK as alternative in caveats

### CODE REVIEW

File: Casks/rubymine-eap.rb [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
 
   app 'RubyMine EAP.app'
+
+  zap :delete => [
+                  '~/Library/Preferences/com.jetbrains.rubymine-EAP.plist',
+                  '~/Library/Preferences/RubyMine70',
+                  '~/Library/Application Support/RubyMine70',
+                  '~/Library/Caches/RubyMine70',
+                  '~/Library/Logs/RubyMine70',
+                  '/usr/local/bin/mine',
```
Comment (radeksimko) [inline Casks/rubymine-eap.rb]: Is this created by the app at some point? If not, we've got `binary` stanza, see [example](https://github.com/caskroom/homebrew-cask/blob/76e341a01e938394e306ffc051cb5b6c7d347816/Casks/terraform.rb#L10-24).

Comment (sebroeder) [inline Casks/rubymine-eap.rb]: > Is this created by the app at some point? If not, we've got binary stanza, see example.

Yes, `/usr/local/bin/mine` is created by RubyMine Initial Configuration when RubyMine is started for the first time.  

screen shot 2015-03-31 at 12 39 47

**Comment by radeksimko** [comment]
Testing this now... :horse_racing:

**Comment by radeksimko** [comment]
Works as expected, thanks. :+1:

**Pull request merged**